src/harness/reverse-proxy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.16-alpine as builder
+
+WORKDIR /app
+COPY . .
+RUN go mod init https://github.com/harness/helm-charts/src/harness/reverse-proxy
```

Comment:
Not sure if this needs tobe modified to reflect branch https://github.com/harness/helm-charts/tree/main/src/harness/reverse-proxy

Reply:
or move reverse-proxy folder under "helm-charts" dir